File: pr1-java/src/main/java/domain/Game.java
package domain;

public class Game extends Entity<Integer>{
    private String name;
    private String homeTeam;
    private String awayTeam;
    private Integer availableSeats;
    private Integer seatCost;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getHomeTeam() {
        return homeTeam;
    }

    public void setHomeTeam(String homeTeam) {
        this.homeTeam = homeTeam;
    }

    public String getAwayTeam() {
        return awayTeam;
    }

    public void setAwayTeam(String awayTeam) {
        this.awayTeam = awayTeam;
    }

    public Integer getAvailableSeats() {
        return availableSeats;
    }

    public void setAvailableSeats(Integer availableSeats) {
        this.availableSeats = availableSeats;
    }

    public Integer getSeatCost() {
        return seatCost;
    }

    public void setSeatCost(Integer seatCost) {
        this.seatCost = seatCost;
    }

    public Game(Integer id, String name, String homeTeam, String awayTeam, Integer availableSeats, Integer seatCost) {
        super(id);
        this.name = name;
        this.homeTeam = homeTeam;
        this.awayTeam = awayTeam;
        this.availableSeats = availableSeats;
        this.seatCost = seatCost;
    }

    @Override
    public String toString() {
        return "Game{" +
                "id='" + getId() + '\'' +
                ", name='" + name + '\'' +
                ", homeTeam='" + homeTeam + '\'' +
                ", awayTeam='" + awayTeam + '\'' +
                ", availableSeats=" + availableSeats + '\'' +
                ", seatCost=" + seatCost +
                '}';
    }
}

File: pr1-cs/Domain/Game.cs
namespace pr1_cs.Domain
{
    public class Game : Entity<int>
    {
        private string _name;
        private string _homeTeam;
        private string _awayTeam;
        private int _availableSeats;
        private int _seatCost;

        public string Name
        {
            get => _name;
            set => _name = value;
        }

        public string HomeTeam
        {
            get => _homeTeam;
            set => _homeTeam = value;
        }

        public string AwayTeam
        {
            get => _awayTeam;
            set => _awayTeam = value;
        }

        public int AvailableSeats
        {
            get => _availableSeats;
            set => _availableSeats = value;
        }

        public int SeatCost
        {
            get => _seatCost;
            set => _seatCost = value;
        }

        public Game(int id, string name, string homeTeam, string awayTeam, int availableSeats, int seatCost) : base(id)
        {
            _name = name;
            _homeTeam = homeTeam;
            _awayTeam = awayTeam;
            _availableSeats = availableSeats;
            _seatCost = seatCost;
        }

        public override string ToString()
        {
		    return "Game{" +
                "id='" + Id+ '\'' +
                ", name='" + Name + '\'' +
                ", homeTeam='" + HomeTeam + '\'' +
                ", awayTeam='" + AwayTeam + '\'' +
                ", availableSeats='" + AvailableSeats + '\'' +
                ", seatCost='" + SeatCost + '\'' +
                '}';
        }
    }
}